apply_migration.py

```
@@ -0,0 +1,98 @@
+        logger.info(f"Reading migration: {mig_file.name}")
+        with open(mig_file, 'r') as f:
+            migration_sql += f"\n-- File: {mig_file.name}\n"
+            migration_sql += f.read()
+            migration_sql += "\n"
+
+    logger.info(f"Applying {len(migration_files)} migration(s)...")
+
+    try:
+        # Execute the migration SQL using the PostgREST SQL endpoint
+        # Note: We need to use the raw SQL execution via RPC
+        result = supabase.rpc('exec_sql', {'sql': migration_sql}).execute()
```

Ruff (0.13.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)